[2021-12-28 13:46:26,631] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_monaghan.scrap_data scheduled__2021-12-27T10:30:00+00:00 [queued]>
[2021-12-28 13:46:26,646] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_monaghan.scrap_data scheduled__2021-12-27T10:30:00+00:00 [queued]>
[2021-12-28 13:46:26,646] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:46:26,647] {taskinstance.py:1242} INFO - Starting attempt 4 of 4
[2021-12-28 13:46:26,648] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:46:26,669] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-27 10:30:00+00:00
[2021-12-28 13:46:26,681] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_monaghan', 'scrap_data', 'scheduled__2021-12-27T10:30:00+00:00', '--job-id', '5111', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_monaghan.py', '--cfg-path', '/tmp/tmpzap12922', '--error-file', '/tmp/tmpzv8hfbef']
[2021-12-28 13:46:26,683] {standard_task_runner.py:77} INFO - Job 5111: Subtask scrap_data
[2021-12-28 13:46:26,676] {standard_task_runner.py:52} INFO - Started process 55557 to run task
[2021-12-28 13:46:26,781] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_monaghan.scrap_data scheduled__2021-12-27T10:30:00+00:00 [running]> on host 0c125a5bba95
[2021-12-28 13:46:26,914] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_monaghan
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-27T10:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-27T10:30:00+00:00
[2021-12-28 13:46:26,935] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-28 13:46:26,938] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py monaghan
[2021-12-28 13:46:28,674] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-28 13:46:28,781] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-28 13:46:28,783] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-28 13:46:28,784] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-28 13:46:28,785] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-28 13:46:28,786] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-28 13:46:28,843] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-28 13:46:28,844] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-28 13:46:28,845] {spark_submit.py:514} INFO - deployMode              null
[2021-12-28 13:46:28,846] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-28 13:46:28,847] {spark_submit.py:514} INFO - executorCores           null
[2021-12-28 13:46:28,847] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-28 13:46:28,848] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-28 13:46:28,849] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-28 13:46:28,849] {spark_submit.py:514} INFO - driverCores             null
[2021-12-28 13:46:28,850] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-28 13:46:28,851] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-28 13:46:28,852] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-28 13:46:28,853] {spark_submit.py:514} INFO - supervise               false
[2021-12-28 13:46:28,854] {spark_submit.py:514} INFO - queue                   null
[2021-12-28 13:46:28,854] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-28 13:46:28,855] {spark_submit.py:514} INFO - files                   null
[2021-12-28 13:46:28,856] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-28 13:46:28,857] {spark_submit.py:514} INFO - archives                null
[2021-12-28 13:46:28,857] {spark_submit.py:514} INFO - mainClass               null
[2021-12-28 13:46:28,858] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-28 13:46:28,859] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-28 13:46:28,860] {spark_submit.py:514} INFO - childArgs               [monaghan]
[2021-12-28 13:46:28,860] {spark_submit.py:514} INFO - jars                    null
[2021-12-28 13:46:28,861] {spark_submit.py:514} INFO - packages                null
[2021-12-28 13:46:28,862] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-28 13:46:28,862] {spark_submit.py:514} INFO - repositories            null
[2021-12-28 13:46:28,863] {spark_submit.py:514} INFO - verbose                 true
[2021-12-28 13:46:28,864] {spark_submit.py:514} INFO - 
[2021-12-28 13:46:28,864] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-28 13:46:28,865] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-28 13:46:28,866] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:46:28,867] {spark_submit.py:514} INFO - 
[2021-12-28 13:46:28,867] {spark_submit.py:514} INFO - 
[2021-12-28 13:46:29,181] {spark_submit.py:514} INFO - Main class:
[2021-12-28 13:46:29,182] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-28 13:46:29,182] {spark_submit.py:514} INFO - Arguments:
[2021-12-28 13:46:29,183] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-28 13:46:29,184] {spark_submit.py:514} INFO - null
[2021-12-28 13:46:29,185] {spark_submit.py:514} INFO - monaghan
[2021-12-28 13:46:29,186] {spark_submit.py:514} INFO - --verbose
[2021-12-28 13:46:29,187] {spark_submit.py:514} INFO - Spark config:
[2021-12-28 13:46:29,188] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-28 13:46:29,188] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:46:29,189] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-28 13:46:29,190] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-28 13:46:29,191] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-28 13:46:29,192] {spark_submit.py:514} INFO - 
[2021-12-28 13:46:29,192] {spark_submit.py:514} INFO - 
[2021-12-28 13:46:29,193] {spark_submit.py:514} INFO - 
[2021-12-28 13:46:30,537] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-28 13:46:30,564] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO SparkContext: Running Spark version 3.2.0
[2021-12-28 13:46:30,680] {spark_submit.py:514} INFO - 21/12/28 13:46:30 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-28 13:46:30,862] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO ResourceUtils: ==============================================================
[2021-12-28 13:46:30,863] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-28 13:46:30,864] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO ResourceUtils: ==============================================================
[2021-12-28 13:46:30,864] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-28 13:46:30,900] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-28 13:46:30,916] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO ResourceProfile: Limiting resource is cpu
[2021-12-28 13:46:30,917] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-28 13:46:30,992] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO SecurityManager: Changing view acls to: ***
[2021-12-28 13:46:30,993] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO SecurityManager: Changing modify acls to: ***
[2021-12-28 13:46:30,994] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO SecurityManager: Changing view acls groups to:
[2021-12-28 13:46:30,995] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO SecurityManager: Changing modify acls groups to:
[2021-12-28 13:46:30,996] {spark_submit.py:514} INFO - 21/12/28 13:46:30 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-28 13:46:31,426] {spark_submit.py:514} INFO - 21/12/28 13:46:31 INFO Utils: Successfully started service 'sparkDriver' on port 42427.
[2021-12-28 13:46:31,472] {spark_submit.py:514} INFO - 21/12/28 13:46:31 INFO SparkEnv: Registering MapOutputTracker
[2021-12-28 13:46:31,535] {spark_submit.py:514} INFO - 21/12/28 13:46:31 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-28 13:46:31,568] {spark_submit.py:514} INFO - 21/12/28 13:46:31 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-28 13:46:31,569] {spark_submit.py:514} INFO - 21/12/28 13:46:31 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-28 13:46:31,577] {spark_submit.py:514} INFO - 21/12/28 13:46:31 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-28 13:46:31,617] {spark_submit.py:514} INFO - 21/12/28 13:46:31 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-c3ee1474-672d-466d-99f8-5662985de848
[2021-12-28 13:46:31,647] {spark_submit.py:514} INFO - 21/12/28 13:46:31 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-28 13:46:31,671] {spark_submit.py:514} INFO - 21/12/28 13:46:31 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-28 13:46:31,952] {spark_submit.py:514} INFO - 21/12/28 13:46:31 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2021-12-28 13:46:32,035] {spark_submit.py:514} INFO - 21/12/28 13:46:32 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://0c125a5bba95:4040
[2021-12-28 13:46:32,328] {spark_submit.py:514} INFO - 21/12/28 13:46:32 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:46:32,435] {spark_submit.py:514} INFO - 21/12/28 13:46:32 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:46:32,436] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:46:32,436] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:46:32,437] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:46:32,438] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:46:32,439] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:46:32,439] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:46:32,440] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:46:32,441] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:46:32,442] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:46:32,443] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:46:32,443] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:46:32,444] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:46:32,445] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:46:32,445] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:46:32,446] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:46:32,446] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:46:32,447] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:46:32,448] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:46:32,448] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:46:32,449] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:46:32,450] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:46:32,450] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:46:32,451] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:46:32,452] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:46:32,452] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:46:32,453] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:46:32,454] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:46:32,454] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:46:32,455] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:46:32,456] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:46:32,456] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:46:32,457] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:46:32,458] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:46:32,458] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:46:32,459] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:46:32,460] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:46:32,460] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:46:32,461] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:46:32,462] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:46:32,463] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:46:32,463] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:46:32,464] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:46:32,465] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:46:32,465] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:46:32,466] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:46:32,467] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:46:32,467] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:46:32,468] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:46:32,469] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:46:32,469] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:46:32,470] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:46:32,471] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:46:32,471] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:46:32,472] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:46:32,473] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:46:32,473] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:46:32,474] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:46:52,292] {spark_submit.py:514} INFO - 21/12/28 13:46:52 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:46:52,306] {spark_submit.py:514} INFO - 21/12/28 13:46:52 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:46:52,307] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:46:52,307] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:46:52,308] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:46:52,309] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:46:52,309] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:46:52,310] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:46:52,311] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:46:52,311] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:46:52,312] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:46:52,313] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:46:52,314] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:46:52,315] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:46:52,315] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:46:52,316] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:46:52,317] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:46:52,318] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:46:52,318] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:46:52,319] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:46:52,319] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:46:52,320] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:46:52,321] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:46:52,322] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:46:52,322] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:46:52,323] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:46:52,324] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:46:52,324] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:46:52,325] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:46:52,326] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:46:52,327] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:46:52,327] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:46:52,328] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:46:52,329] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:46:52,329] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:46:52,330] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:46:52,331] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:46:52,331] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:46:52,332] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:46:52,333] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:46:52,333] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:46:52,334] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:46:52,335] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:46:52,336] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:46:52,337] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:46:52,338] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:46:52,338] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:46:52,339] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:46:52,340] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:46:52,340] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:46:52,341] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:46:52,342] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:46:52,342] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:46:52,343] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:46:52,344] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:46:52,345] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:46:52,345] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:46:52,346] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:46:52,347] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:47:12,259] {spark_submit.py:514} INFO - 21/12/28 13:47:12 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:47:12,275] {spark_submit.py:514} INFO - 21/12/28 13:47:12 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:47:12,276] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:47:12,277] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:47:12,278] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:47:12,279] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:47:12,280] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:47:12,280] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:47:12,281] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:47:12,282] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:47:12,283] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:47:12,284] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:47:12,285] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:47:12,286] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:47:12,288] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:47:12,288] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:47:12,289] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:47:12,290] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:47:12,291] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:47:12,292] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:47:12,292] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:47:12,293] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:47:12,294] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:47:12,295] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:47:12,295] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:47:12,296] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:47:12,297] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:47:12,297] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:47:12,298] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:47:12,299] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:47:12,300] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:47:12,300] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:47:12,301] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:47:12,302] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:47:12,303] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:47:12,303] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:47:12,304] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:47:12,305] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:47:12,305] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:47:12,306] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:47:12,307] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:47:12,307] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:47:12,308] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:47:12,309] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:47:12,310] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:47:12,310] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:47:12,312] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:47:12,312] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:47:12,313] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:47:12,314] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:47:12,314] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:47:12,315] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:47:12,316] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:47:12,316] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:47:12,317] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:47:12,318] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:47:12,318] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:47:12,319] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:47:12,320] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:47:32,260] {spark_submit.py:514} INFO - 21/12/28 13:47:32 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2021-12-28 13:47:32,262] {spark_submit.py:514} INFO - 21/12/28 13:47:32 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2021-12-28 13:47:32,280] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 37861.
[2021-12-28 13:47:32,281] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO NettyBlockTransferService: Server created on 0c125a5bba95:37861
[2021-12-28 13:47:32,282] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-28 13:47:32,293] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 0c125a5bba95, 37861, None)
[2021-12-28 13:47:32,300] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO BlockManagerMasterEndpoint: Registering block manager 0c125a5bba95:37861 with 434.4 MiB RAM, BlockManagerId(driver, 0c125a5bba95, 37861, None)
[2021-12-28 13:47:32,305] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 0c125a5bba95, 37861, None)
[2021-12-28 13:47:32,311] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 0c125a5bba95, 37861, None)
[2021-12-28 13:47:32,319] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO SparkUI: Stopped Spark web UI at http://0c125a5bba95:4040
[2021-12-28 13:47:32,339] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-28 13:47:32,359] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-28 13:47:32,381] {spark_submit.py:514} INFO - 21/12/28 13:47:32 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2021-12-28 13:47:32,412] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-28 13:47:32,454] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO MemoryStore: MemoryStore cleared
[2021-12-28 13:47:32,455] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO BlockManager: BlockManager stopped
[2021-12-28 13:47:32,473] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-28 13:47:32,483] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-28 13:47:32,506] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO SparkContext: Successfully stopped SparkContext
[2021-12-28 13:47:32,612] {spark_submit.py:514} INFO - 21/12/28 13:47:32 ERROR SparkContext: Error initializing SparkContext.
[2021-12-28 13:47:32,613] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:47:32,615] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:47:32,615] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:47:32,616] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:47:32,618] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:47:32,618] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:47:32,619] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:47:32,620] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:47:32,621] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:47:32,622] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:47:32,623] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:47:32,624] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:47:32,624] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:47:32,625] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:47:32,626] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:47:32,627] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:47:32,627] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:47:32,628] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO SparkContext: SparkContext already stopped.
[2021-12-28 13:47:32,629] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-28 13:47:32,630] {spark_submit.py:514} INFO - File "/opt/***/jobs/get_ads_per_county.py", line 13, in <module>
[2021-12-28 13:47:32,630] {spark_submit.py:514} INFO - .appName("Get Ads per county") \
[2021-12-28 13:47:32,631] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 228, in getOrCreate
[2021-12-28 13:47:32,632] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 392, in getOrCreate
[2021-12-28 13:47:32,632] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 147, in __init__
[2021-12-28 13:47:32,634] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2021-12-28 13:47:32,634] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2021-12-28 13:47:32,635] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/java_gateway.py", line 1574, in __call__
[2021-12-28 13:47:32,636] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/protocol.py", line 328, in get_return_value
[2021-12-28 13:47:32,637] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2021-12-28 13:47:32,638] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:47:32,638] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:47:32,639] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:47:32,640] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:47:32,641] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:47:32,641] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:47:32,642] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:47:32,643] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:47:32,644] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:47:32,644] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:47:32,645] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:47:32,646] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:47:32,647] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:47:32,648] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:47:32,648] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:47:32,649] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:47:32,650] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:47:32,651] {spark_submit.py:514} INFO - 
[2021-12-28 13:47:32,711] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO ShutdownHookManager: Shutdown hook called
[2021-12-28 13:47:32,714] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO ShutdownHookManager: Deleting directory /tmp/spark-71742851-7ecc-4055-aab1-a8dc81625402
[2021-12-28 13:47:32,719] {spark_submit.py:514} INFO - 21/12/28 13:47:32 INFO ShutdownHookManager: Deleting directory /tmp/spark-775bacfe-524b-4ee7-a436-895c4165389f
[2021-12-28 13:47:32,838] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py monaghan. Error code is: 1.
[2021-12-28 13:47:32,845] {taskinstance.py:1280} INFO - Marking task as FAILED. dag_id=scraping_daft_monaghan, task_id=scrap_data, execution_date=20211227T103000, start_date=20211228T134626, end_date=20211228T134732
[2021-12-28 13:47:32,909] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/utils/email.py:102: PendingDeprecationWarning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=smtp_mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2021-12-28 13:47:32,910] {email.py:208} INFO - Email alerting: attempt 1
[2021-12-28 13:47:34,004] {email.py:220} INFO - Sent an alert email to ['clement.liscoet@gmail.com']
[2021-12-28 13:47:35,981] {standard_task_runner.py:91} ERROR - Failed to execute job 5111 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py monaghan. Error code is: 1.
[2021-12-28 13:47:36,021] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-12-28 13:47:36,107] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
